exercises/ocr-numbers/example.py

```diff
@@ -16,28 +16,25 @@ def split_ocr(ocr):
 OCR_LIST = [OCR_LIST[-1]] + OCR_LIST[:9]
 
 
-def number(ocr):
-    if (len(ocr) != ROW or len(ocr[0]) % COL or
-            any(len(r) != len(ocr[0]) for r in ocr)):
+def convert(input_grid):
```

Comment:
Exercise placeholder should be also updated in this case:
https://github.com/exercism/python/blob/master/exercises/ocr-numbers/ocr_numbers.py